Engines/Wine/Verbs/dotnet20sp2/script.js

```diff
@@ -0,0 +1,76 @@
+include("engines.wine.engine.object");
+include("engines.wine.plugins.override_dll");
+include("utils.functions.net.resource");
+include("utils.functions.filesystem.files");
+include("engines.wine.plugins.windows_version");
+include("engines.wine.verbs.remove_mono");
+
+
+/**
+* Verb to install .NET 2.0 SP2
+* @returns {Wine} Wine object
+*/
+Wine.prototype.dotnet20sp2 = function () {
+    var osVersion = this.windowsVersion();
+    this.windowsVersion("winxp");
+    var dlls = [
```

Comment:
On one line, and I think you should remove those dll overrides at the end (that is why winetricks use an env var).

Reply:
vcrun2003 also have those dll overrides on separate lines

Reply:
Ok, but I find this ugly ^^.

Reply:
And why do I need to remove the dll overrides? It's not how winetricks does this.

Reply:
Winetricks use an environment variable to only change the dlls while installing the verb.

Reply:
Will the latest commit suffice?